src/main/java/players/mcts/MCTSSearchSpace.java

```diff
@@ -0,0 +1,135 @@
+/*
```

Comment:
Ugly to have a class that's all commented out, maybe keep it local and put it back in the branch after merge if needed?

Reply:
Oops. That was my mistake. I just forgot to delete it once I was happy I hadn't missed anything.